sourcemash/frontend/static/js/views/browse_view.js

```diff
@@ -40,10 +47,16 @@ Sourcemash.Views.BrowseView = Backbone.View.extend({
     this.loadingView = new Sourcemash.Views.LoadingView({loading: this.loading});
     this.$(".loading").html(this.loadingView.render().el);
 
+    // Render register modal view
+    this.registerModalView = new Sourcemash.Views.RegisterModalView();
+    this.$("#register-modal").html(this.registerModalView.render().el);
+
     // Render item cards
     var feedCards = [];
+    _user = this.user;
```

Comment:
Would that require writing it as it's own function and calling it here then? Is that necessary?